src/components/Book.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({
+  book: {
+    id,
+    title,
+    category,
+  },
+}) {
```

Comment:
Since you are already destructuring the props, including `book` in your props is redundant. It is better to express your function this way:
```suggestion
function Book({
    id,
    title,
    category
}) {
```

Reply:
This is resolved well